www/js/messages.js

```diff
@@ -215,18 +215,22 @@ function initMessages(head, query) {
     }
   });
 
-  window.addEventListener('native.keyboardshow', function(e) {
-    F7msg.scrollMessages(350);
+  window.addEventListener('native.keyboardshow', function() {
+    if (F7msg) {
```

Comment:
The events don't appear to be removed when we close the screen in v2, so we need to check if F7 is def'ed to avoid getting errors on other input fields.